From 80cf2b9462535e5265c41d0b0c97f33df835d786 Thu Oct 23 13:49:27 2025
From: Serhii Kupriienko
Date: Thu, 23 Oct 2025 13:49:27 +0300
Subject: [PATCH] Fix pyproject.toml license format and python version
 constraint

---
 pyproject.toml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 1e24356..eeb1375 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@
 
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools>=77", "Cython", "numpy"]
+requires = ["setuptools>=64", "Cython", "numpy"]
 
 [project]
 authors = [
@@ -56,10 +56,10 @@ dynamic = ["version"]
 keywords = ["MKL", "VSL", "true randomness", "pseudorandomness",
             "Philox", "MT-19937", "SFMT-19937", "MT-2203", "ARS-5",
             "R-250", "MCG-31"]
-license = "BSD-3-Clause"
+license = {text = "BSD-3-Clause"}
 name = "mkl_random"
 readme = {file = "README.md", content-type = "text/markdown"}
-requires-python = ">=3.9,<3.14"
+requires-python = ">=3.9,<3.15"
 
 [project.optional-dependencies]
 test = ["pytest"]
-- 
2.50.1 (Apple Git-155)

